docs/permissions/plugin-authors/01-setup.md

```diff
@@ -26,6 +26,9 @@ The source code is available here:
       git checkout master -- plugins/example-todo-list/
       git checkout master -- plugins/example-todo-list-backend/
       git checkout master -- plugins/example-todo-list-common/
+      sed -i '' 's/workspace:\^/\*/g' plugins/example-todo-list/package.json
+      sed -i '' 's/workspace:\^/\*/g' plugins/example-todo-list-backend/package.json
+      sed -i '' 's/workspace:\^/\*/g' plugins/example-todo-list-common/package.json
```

Comment:
Why modify these files after instead of just putting `*` into each file?

Reply:
> Why modify these files after instead of just putting `*` into each file?

because the dependencies live in the same monorepo, as in [here](https://github.com/backstage/backstage/blob/master/plugins/example-todo-list/package.json#L27)